docs/source/json-api/lf-value-specification.rst

```diff
@@ -213,9 +213,10 @@ quotes::
     "Bob"
 
 Unit
-----
+****
 
 Represented as empty object ``{}``. Note that in JavaScript ``{} !==
```

Comment:
The backtick literals in this paragraph that are split across lines probably need to be joined back to start and end in the same line.

Reply:
Ah, I just accidentally added an empty line between them; renders fine after reverting that d2ef790834